src/duplicacy_chunk.go

```diff
@@ -68,11 +70,13 @@ type Chunk struct {
 }
 
 // Magic word to identify a duplicacy format encrypted file, plus a version number.
-var ENCRYPTION_HEADER = "duplicacy\000"
+var ENCRYPTION_BANNER = "duplicacy\000"
 
 // RSA encrypted chunks start with "duplicacy\002"
 var ENCRYPTION_VERSION_RSA byte = 2
 
+var ERASURE_CODING_BANNER = "duplicacy\003"
```

Comment:
I could be missunderstanding stuff but
`RSA encrypted chunks start with "duplicacy\002"`
`ERASURE_CODING_BANNER = "duplicacy\003"`
does this mean that we cannot have rsa and erasure at the same time?

Reply:
If we can have all of those, then the flag `ENCRYPTION_BANNER ` could be better renamed to `FEATURE_LEVEL`.